src/test/common.go

```diff
@@ -89,7 +97,10 @@ func (e2e *ZarfE2ETest) Kubectl(t *testing.T, args ...string) (string, string, e
 // CleanFiles removes files and directories that have been created during the test.
 func (e2e *ZarfE2ETest) CleanFiles(files ...string) {
 	for _, file := range files {
-		_ = os.RemoveAll(file)
+		err := os.RemoveAll(file)
```

Comment:
Would we run into issues if we did a `require.NoError` here instead? Given that [removeAll](https://pkg.go.dev/os#RemoveAll) returns nil when a file doesn't exist I would think we wouldn't, but I have not tried it out.

Reply:
That would be ideal, but there wasn't testing.T state here. I'll look into passing it through